src/sfdc/salesforceBulkUtils/sf_bulk_utils.bal

```diff
@@ -136,48 +124,42 @@ function getQueryRequest(http:Response|error httpResponse, JOBTYPE jobtype) retu
     }
 }
 
-# Handle HTTP error response and return HttpResponseHandlingError error.
+# Handle HTTP error response and return error.
 # + httpResponse - error response
-# + return - HttpResponseHandlingError error
-function handleXmlErrorResponse(http:Response httpResponse) returns @tainted HttpResponseHandlingError {
+# + return - error
+function handleXmlErrorResponse(http:Response httpResponse) returns @tainted Error {
     xml|error xmlResponse = httpResponse.getXmlPayload();
     xmlns "http://www.force.com/2009/06/asyncapi/dataload" as ns;
 
     if (xmlResponse is xml) {
-        HttpResponseHandlingError httpResponseHandlingError = error(HTTP_RESPONSE_HANDLING_ERROR,
-            message = (xmlResponse/<ns:exceptionMessage>/*).toString(), 
+        Error httpResponseHandlingError = Error((xmlResponse/<ns:exceptionMessage>/*).toString(), 
             errorCode = (xmlResponse/<ns:exceptionCode>/*).toString());
         return httpResponseHandlingError;
     } else {
         log:printError(ERR_EXTRACTING_ERROR_MSG, err = xmlResponse);
-        HttpResponseHandlingError httpResponseHandlingError = error(HTTP_RESPONSE_HANDLING_ERROR,
-            message = ERR_EXTRACTING_ERROR_MSG, errorCode = HTTP_RESPONSE_HANDLING_ERROR, cause = xmlResponse);
-        return httpResponseHandlingError;
+        return Error(ERR_EXTRACTING_ERROR_MSG, xmlResponse);
     }
 }
 
-# Handle HTTP error response and return HttpResponseHandlingError error.
+# Handle HTTP error response and return Error.
 # + httpResponse - error response
-# + return - HttpResponseHandlingError error
-function handleJsonErrorResponse(http:Response httpResponse) returns @tainted HttpResponseHandlingError {
+# + return - error
+function handleJsonErrorResponse(http:Response httpResponse) returns @tainted Error {
     json|error response = httpResponse.getJsonPayload();
     if (response is json) {
-        HttpResponseHandlingError httpResponseHandlingError = error(HTTP_RESPONSE_HANDLING_ERROR,
-            message = response.exceptionMessage.toString(), errorCode = response.exceptionCode.toString());
+        Error httpResponseHandlingError = Error(response.exceptionMessage.toString(), errorCode = response.exceptionCode.toString());
```

Comment:
Is `errorCode` still valid. since we are not using `ErrorDetail` record now

Reply:
Shall we remove passing `errorCode` param. Since we don't have `ErrorDetail` record. passing the value will just skip

Reply:
done